### PR TITLE
Box folders

### LINKED ISSUE #185: Box folders
Currently, all files uploaded to Box go to the root folder. However, as box does not accept files with the same name, this means attempting to upload a file when another file with the same name has been uploaded will not work.

Folders will be allocated to each portfolio manager, and within each portfolio manager folder more folders will be allocated to each field partner. Folders will be created when a user is created and documents will be uploaded to the appropriate field partner folder. If a document is uploaded when one with the same name exists, it will update the existing version instead.

### PR DESCRIPTION
**Pull request by arpanlaha**
Resolves #185

### CODE REVIEW

File: backend/api/views/box.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def upload_file(file, file_name):
     stream.write(file.read())
     stream.seek(0)
     try:
-        box_file = client.folder("0").upload_stream(
+        box_file = client.folder(str(folder_id)).upload_stream(
```
Comment (choiboy98) [inline backend/api/views/box.py]: Do you have to cast it into a str()? Isn't the parameter a string already?

Comment (choiboy98) [reply backend/api/views/box.py]: same goes to other places that use str()

**Comment by rgychiu** [comment]
Might want to check if viewing a file still works as well, not sure if that's just something on my side (viewing both in modal as well as going to the page itself since now the files are in folders)

File: backend/api/views/box.py (hunk 1 above)
Comment (arpanlaha) [reply backend/api/views/box.py]: just to be safe

**Pull request merged**